dataactcore/models/views.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import MetaData, Table
+from dataactcore.interfaces.db import GlobalDB
+
+
+class SubmissionUpdatedView:
```

Comment:
I personally don't. Just wasn't sure what the better route it. We can leave it the way it is for now and update if we come across something that clearly defines if one of those is best practice.